pilot/pkg/networking/core/v1alpha3/tls.go

```diff
@@ -247,10 +235,11 @@ func buildSidecarOutboundTCPFilterChainOpts(node *model.Proxy, env *model.Enviro
 	return out
 }
 
-func buildSidecarOutboundTCPTLSFilterChainOpts(node *model.Proxy, env *model.Environment, configs []model.Config, destinationIPAddress string,
-	service *model.Service, listenPort *model.Port, proxyLabels model.LabelsCollection, gateways map[string]bool) []*filterChainOpts {
+func buildSidecarOutboundTCPTLSFilterChainOpts(node *model.Proxy, env *model.Environment, destinationIPAddress string,
+	service *model.Service, listenPort *model.Port, proxyLabels model.LabelsCollection, gateways map[string]bool,
+	byHost virtualServiceForHost) []*filterChainOpts {
```

Comment:
time to make this a context variable where we stuff all args?

Reply:
...maybe.